scripts/dream.py

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python3
 # Copyright (c) 2022 Lincoln D. Stein (https://github.com/lstein)
 
-import sys
-import os.path
-
-script_path = sys.argv[0]
-script_args = sys.argv[1:]
-script_dir,script_name = os.path.split(script_path)
-script_dest = os.path.join(script_dir,'invoke.py')
-os.execlp('python3','python3',script_dest,*script_args)
+import warnings
+import invoke
 
+if __name__ == '__main__':
+    warnings.warn("dream.py is deprecated, please run invoke.py instead",
```

Comment:
We could mention the newly merged `legacy api` here which contains the old API interface.

So something like "dream.py is being deprecated, please run `invoke.py` for the new UI/API or `legacy_api.py` for the old API"

Reply:
Modified as per above